library/ssl_tls.c

```diff
@@ -1925,6 +1936,7 @@ mbedtls_ssl_mode_t mbedtls_ssl_get_mode_from_ciphersuite(
  *       uint32 ticket_age_add;
  *       uint8 ticket_flags;
  *       opaque resumption_key<0..255>;
+ *       opaque hostname<1..2^8-1>;
```

Comment:
```suggestion
 *       opaque hostname<1..2^16-1>;
```

From https://datatracker.ietf.org/doc/html/rfc6066#section-3 , `opaque HostName<1..2^16-1>;`